cmd/kubeadm/app/preflight/checks.go

```diff
@@ -358,15 +360,19 @@ func (ipc InPathCheck) Name() string {
 }
 
 // Check validates if the given executable is present in the path.
-func (ipc InPathCheck) Check() (warnings, errors []error) {
+func (ipc InPathCheck) Check() (warnings, errs []error) {
 	_, err := ipc.exec.LookPath(ipc.executable)
 	if err != nil {
 		if ipc.mandatory {
 			// Return as an error:
 			return nil, []error{fmt.Errorf("%s not found in system path", ipc.executable)}
 		}
 		// Return as a warning:
-		return []error{fmt.Errorf("%s not found in system path", ipc.executable)}, nil
+		errorMessage := fmt.Sprintf("%s not found in system path", ipc.executable)
```

Comment:
This is technically a warning message, since it's being returned in the first slice and not the second. Not sure if that's too nit-picky :)

Reply:
+1